### DISCUSSION TITLE
Our first discussion - Thảo luận đầu tiên của nhóm

### DISCUSSION THREAD

**Discussion by VOM7HC** [vi-vn-cppreference/vietnamese-cppreference#3] [Announcements]
@tongtunggiang @thangnguyenmd @nguyenphuc241 @nguyenntdev @ndhoang123 @mariejie @khartist @hoanhdung3 @hangnguyenthi20 

Chia nhóm các bạn react bằng emoji:

👍 nhóm 1

😄 nhóm 2

[English] Dear my teammate,

This is the our first discussion for a open source project, a translation [C++ Reference](https://en.cppreference.com/w/) from English to Vietnamese. I very appreciate your willing to help to grow our community. For who has experience, this will help your revise old knowledge, maybe have job opportunities on your next step career. For who just begin a path to be software engineer, this will a vista what would you do in the future.

For clearing our project, I just want to hearing your voice in some sectors

- As what I talked before, I will divide our team in two groups. Group 1 for who has experience professional working with C/C++ or native language like Assembly/Golang/Rust or junior student has experience in use C/C++ . Group 2 for who is student (sophomore year below) or has experience professional working is other programming language.

- For our group chatting, do you like GitHub discussion or using Slack, Team. I recommend and prefer not to use Facebook, because I thought everyone has their privacy.

- What would you want to use? HTML cppreference website or Markdown GitHub wiki?

- I will create a task. Do you want to use Trello or [GitHub projects ](https://github.com/vi-vn-cppreference/vietnamese-cppreference/projects/1) (a kanban board). I created a GitHub projects board as sample for us.

- If you can't attend in group maybe one day in future, you can ask me or future group leader.

 I love to hear more discuss from your side to keep our community grower. Many thanks.
 
Your sincerely.

[Tiếng Việt] Thân gửi các bạn,

Đây sẽ là cuộc thảo luận đầu tiên của chúng ta cho dự án mã nguồn mở, dự án chuyển ngữ website [C++ Reference](https://en.cppreference.com/w/) từ tiếng Anh sang tiếng Việt. Mình rất cảm kích với sự sẵn lòng giúp đỡ cho dự án trở nên lớn mạnh hơn. Với những bạn có kinh nghiệm, dự án sẽ giúp bạn ôn lại và nắm vững kiến thức hơn cho con đường phát triển sự nghiệp sau này. Với những bạn đang bước vào con đường trở thành lập trình viên, dự án sẽ là tầm nhìn về những việc bạn sẽ làm trong tương lai.

Để dự án phát triển hơn mình mong mình cần thu thập các ý kiến từ các bạn

-  Như đã thảo luận từ trước. Mình sẽ chia làm 2 nhóm. Nhóm 1 các bác có kiến thức về C/C++ hoặc là ngôn ngữ như Assembly/Golang/Rust hoặc sinh viên năm 3 thường dùng C/C++ là ngôn ngữ chính. Nhóm 2 các bác có kinh nghiệm ở những ngôn ngữ lập trình khác hoặc sinh viên năm thứ 2.

-  Để nói chuyện bạn muốn xài dịch vụ nào GitHub discussion này hay Slack hay Team? Mình thì khuyến khích không dùng và không muốn dùng Facebook vì chúng ta cần giữ cuộc sống cá nhân của mỗi thành viên.

-  Bạn muốn chuyển ngữ bằng cách chỉnh trực tiếp file HTML hay xài GitHub wiki dạng Markdown?

-  Với phân chia công việc dịch mỗi bài, các bạn thích dùng Trello hay [GitHub projects](https://github.com/vi-vn-cppreference/vietnamese-cppreference/projects/1) (bảng kanban). Mình tạo mẫu sẵn 1 cái GitHub project, các bạn có thể tham khảo.

- Nếu một ngày nào đó, bạn không thể tham gia group được nữa, có thể nhắn cho mình hoặc trưởng nhóm khác trong tương lai.

Mình rất muốn nghe thêm các ý kiến của các bạn để cộng đồng ngày càng phát triển hơn.

Chân thành cảm ơn các bạn.

**Comment by ttsoncs** [comment]
Theo mình thì github markdown, giống core guidelines, dễ đọc, sạch đẹp.

**Comment by ndhoang123** [comment]
> Để nói chuyện bạn muốn xài dịch vụ nào GitHub discussion này hay Slack hay Team? 

- Ý này thì mình thấy là nếu thảo luận thông thường thì dùng Github discussion cũng được, để tránh trường hợp bị trôi thông tin mà các bạn sau không nắm bắt được. Còn nếu mà trao đổi về công việc, như dịch thuật các thứ, thì dùng Slack, hoặc Team sẽ tiện hơn. Và mình cũng đồng ý là không dùng fb.

> các bạn thích dùng Trello hay [GitHub projects](https://github.com/vi-vn-cppreference/vietnamese-cppreference/projects/1) (bảng kanban).

Còn ý này thì mình nghĩ cái nào tiện cho mọi người thì dùng. Mà mình nghĩ là các công ty dùng Trello, thì mình thấy dùng hẳn Trello cho tiện luôn.

> Bạn muốn chuyển ngữ bằng cách chỉnh trực tiếp file HTML hay xài GitHub wiki dạng Markdown?

Theo mình thấy nên dùng Markdown hơn.

**Comment by pham304** [comment]
Nên sử dụng một trình biên tập WYSIWYG, có thể là Markdown hoặc HTML, nhưng cần giữ nguyên cấu trúc thư mục (đường dẫn, category) của CPP Reference để tiện tra cứu.

**Comment by pham304** [comment]
P/s: Để có trí tuệ cộng đồng thì các bài viết nên để public, bất kì ai đều có quyền sửa, qua tối thiểu 1 reviewer thì cho phép commit vào repo chính (dạng wiki).

**Comment by tongtunggiang** [comment]
Mình nghĩ là có những thảo luận nên để ở các kênh riêng tư hơn như Slack/Discord/Teams/ thay vì public như Github Discussion, vì bản chất của chúng khác nhau.

**Comment by hangnguyenthi20** [comment]
Cá nhân mình thích dùng Trello và thảo luận qua Discord vì 2 cái này vừa trông dễ hiểu, dễ làm việc và tiện. Không biết mọi người thế nào

**Comment by khartist** [comment]
Cá nhân em thích sử dụng Slack. Dễ trao đổi thông tin công việc hơn Discord

**Comment by VOM7HC** [comment]
Welcome thêm bác @kensang248 mới vào, bác vào vote luôn nhé

**Comment by VOM7HC** [comment]
Buổi thảo luận thứ hai: https://github.com/vi-vn-cppreference/vietnamese-cppreference/discussions/5
